package.json

```
@@ -7,7 +7,7 @@
     "build": "node lib/build",
     "pdf": "node lib/build-pdf.mjs",
     "start": "node lib/server",
-    "test": "c8 -r html -r text mocha"
+    "test": "c8 -r html -r text mocha --timeout 5000"
```

```suggestion
    "test": "c8 -r html -r text mocha --timeout 10000"
```
First test run on my machine caused a timeout with puppeteer and the old setting